public/locales/es/graph.json

```diff
@@ -0,0 +1,10 @@
+{
+    "Monday":"Lu",
```

Comment:
Maybe use these abbreviations instead, from [Wikipedia](https://en.wikipedia.org/wiki/Date_and_time_notation_in_Spain#:~:text=In%20Spanish%2C%20there%20exists%20a,D%20means%20domingo%20(Sunday).)

Reply:
Agree. While I hadn't seen those abbreviations Carson found, they seem less awkward then these